testsuite/integration/elytron/src/test/resources/arquillian.xml

```diff
@@ -13,7 +13,7 @@
         <configuration>
             <property name="javaVmArguments">${server.jvm.args}</property>
             <property name="serverConfig">${jboss.server.config.file.name:standalone.xml}</property>
-            <property name="jbossArguments">${jboss.args}</property>
+            <property name="jbossArguments">${jboss.args} --stability=${embedded.server.stability}</property>
```

Comment:
And then this would be `${embedded.stability}`

Reply:
fixed!